uktrade/data-hub-frontend
# New dashboard: reminders component

## Description of change

Adds the new reminders component to the dashboard, displaying outstanding propositions and their deadline dates, with a link back to the project propositions page.

A notification badge component was also added to show the number of reminders - this is based on the [MOJ Notification Badge](https://moj-design-system.herokuapp.com/components/notification-badge).

These components have also been added to storybook.

## Test instructions

With the feature flag set for the new dashboard page, visit the dashboard. The reminders component should appear in the top left corner. Check that the links go to the investment project propositions pages.

Also check that the `NotificationBadge` and `InvestmentReminders` appear correctly in storybook.

## Screenshots

![Screenshot from 2021-03-11 11-19-27](https://user-images.githubusercontent.com/1234577/110779684-b974b380-825b-11eb-8d2c-0822ebcca384.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
